Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.module.scss

```diff
@@ -40,12 +40,16 @@
     flex: 1 0 6vw;
   }
 
-  .dateAndUsernameContainer {
+  .IDAndDateAndUsernameContainer {
```

Comment:
Change name to IdAndDateAndUsernameContainer. There are too many uppercase letters and this lowers readability.